app/main.py

```diff
@@ -1,3 +1,89 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int) -> Distance:
+        if isinstance(other, Distance):
+            distance_to_add = other.km
+        else:
+            distance_to_add = other
+        return Distance(
+            self.km + distance_to_add
+        )
+
+    def __iadd__(self, other: Distance | int) -> Distance:
+        if isinstance(other, Distance):
+            distance_to_add = other.km
+        else:
+            distance_to_add = other
+        self.km += distance_to_add
+        return self
+
+    def __mul__(self, other: int) -> Distance:
+        return Distance(
+            km=self.km * other
+        )
+
+    def __truediv__(self, other: int) -> Distance:
+        return Distance(
+            km=round(self.km / other, 2)
+        )
+
+    def __lt__(self, other: Distance) -> bool:
+        if isinstance(other, Distance):
+            distance_to_compare = other.km
```

Comment:
You can return comparison inside if else it will be the same as returning a bool:

```
        if isinstance(other, Distance):
            return self.km < other.km
        else:
            return self.km < other
```